### DISCUSSION TITLE
How to un-blacklist an item?

### DISCUSSION THREAD

**Discussion by Reioflight** [NillerMedDild/Enigmatica6#2834] [Q&A]
I don't know how to code, so I do not feel comfortable modifying anything without help.

`blackList = ["losttrinkets:tha_ghost"]`

At the top of the Lost Trinkets config, there is this sentence:

`#Set to false to disable the default way of unlocking trinkets.
unlockEnabled = true`

So do I change it to    blackList = ["losttrinkets:tha_ghost"][false]    or something?

**Comment by Darkere** [comment]
Those are 2 different unrelated options. The first one is a blacklist to disable certain trinkets. The other decides whether you unlock trinkets by doing regular minecraft tasks.

**Comment by NielsPilgaard** [comment]
You change it to `blackList = []` :)
